packages/textlint/src/util/find-util.ts

```diff
@@ -46,11 +76,14 @@ export function findFiles(patterns: string[], options: { cwd?: string } = {}): s
             addFile(fs.realpathSync(file));
         } else {
             glob.sync(pattern, {
-                nodir: true
+                cwd,
+                absolute: true,
+                nodir: true,
+                ignore: ignoredPatterns
             }).forEach((filePath: string) => {
                 // workaround for windows
                 // https://github.com/isaacs/node-glob/issues/74#issuecomment-31548810
-                addFile(path.resolve(filePath));
+                addFile(path.resolve(cwd, filePath));
```

Comment:
[It is just a question. not need to change]

Is this `cwd` needed?
`absolute` option makes `filePath` to absolute path.
So, I think that `cwd` is always just ignored.

Reply:
You are quite right, `cwd` is no longer needed.
I removed it.